### ISSUE TITLE
Cannot build project when swiftlint 0.43.0+ is installed

### ISSUE THREAD

**Issue by sebskuse** [DiUS/pact-consumer-swift#120]
Hi,

We're currently integrating `pact-consumer-swift` via Carthage. Recently we've noticed that this has started to fail to build via Carthage, or indeed via Xcode directly. We're getting a build error during the `Swiftlint` build phase:

```
Linting Swift files in current working directory
Error: No lintable files found at paths: ''
```

Looking at the `swiftlint` changelog (https://github.com/realm/SwiftLint/releases) for 0.43.0 shows a change was made related to paths inside configuration files:

> Configuration files now consistently have their included/excluded
relative file paths applied relative to their location in the file
system. Previously the root configuration file applied these relative
to the current working directory, but nested configurations applied
these to their location in the file system.

Because the swiftlint configuration file resides within `Configuration`, the `included` directory of `- Sources/` is being interpreted as `./Configuration/Sources/` rather than `./Sources/`, so is unable to lint anything and fails the build. This can be validated by prefixing paths in `included` and `excluded` with `../` which then lints all the expected files and builds successfully.

A few different fixes here, but possibly moving the configuration back to `.swiftlint.yml` in the root of the directory is the easiest one that maintains backwards compatibility with the older versions of Swiftlint?

**Comment by surpher** [comment]
Thanks for raising this. 

Personally I have nothing against moving it back to root of this project. Especially when it's (still) the only file in `/Configuration` folder. Will need an update of a swiftlint sh script. PRs welcome 👍

**Comment by sebskuse** [comment]
Closed with #121
